### DISCUSSION TITLE
Trying to get recognize to work on nextcloud aio

### DISCUSSION THREAD

**Discussion by terrorbyte704** [nextcloud/all-in-one#4675] [Questions] [answered]
I Downloaded the machine learning models, and added the these lines to my docker compose as it said in the github but am getting this error and not sure how to solve it "Could not execute the Node.js binary. You may need to set the path to a working binary manually" any help?
- type: tmpfs
        target: /tmp:exec

**Comment by szaimen** [comment, marked as answer]
> "Could not execute the Node.js binary. You may need to set the path to a working binary manually"

Apparently this id what you need to do
